src/main/java/md2k/mcerebrum/cstress/features/ECGDataQuality.java

```diff
@@ -50,8 +69,22 @@ public ECGDataQuality(DataStreams datastreams, double qualityThreshold) {
         DataPointStream ecg = datastreams.getDataPointStream(StreamConstants.ORG_MD2K_CSTRESS_DATA_ECG);
         DataPointStream ecgQuality = datastreams.getDataPointStream(StreamConstants.ORG_MD2K_CSTRESS_DATA_ECG_QUALITY);
 
-        ECGQualityCalculation ecgComputation = new ECGQualityCalculation(3, 50, 4500, 20, 2, 47);
-        List<DataPoint> quality = ecgComputation.computeQuality(ecg.data, 5000); //0.67
+        this.acceptableOutlierPercent = AUTOSENSE.QUALITY_acceptableOutlierPercent;
```

Comment:
Refactor to just call the constants where needed.  There is no need to introduce additional variables to do this